docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -361,3 +361,16 @@ type:: string
 default:: `appuio.io/active-deadline-seconds-override`
 
 The key of the namespace annotation which users can use to override the global default value for `.spec.activeDeadlineSeconds`.
+
+=== `runOnceActiveDeadlineSeconds.podMatchExpressions`
+
+[horizontal]
+type:: dict
+default:: `{}`
+
+The entries of the dict are expected to be of form `labelKey: <partial matchExpression>`.
+Each entry is transformed to a Kubernetes `matchExpression` entry by taking the key as value for field `name` of the resulting match expression.
+
```

Comment:
This description is a bit unclear to me (I would be looking at code/tests how to configure it). Maybe add the cert-manager example that is in the `tests/defaults.yml`?

(note: I'm not sure if the code suggestion renders nicely)
```suggestion
example::
[source,yaml]
----
podMatchExpressions:
  acme.cert-manager.io/http01-solver:
    operator: DoesNotExist
----

The entries of the dict are expected to be of form `labelKey: <partial matchExpression>`.
Each entry is transformed to a Kubernetes `matchExpression` entry by taking the key as value for field `name` of the resulting match expression.

```

Reply:
Added the example in the latest commit.